### DISCUSSION TITLE
Passing a number as a string to a string metadata type issue

### DISCUSSION THREAD

**Discussion by FamedBear16** [mProjectsCode/obsidian-meta-bind-plugin#483] [Q&A] [answered]
Hi All,

I have a note with a string _status_ metadata field..

I need to store in is **status: "2001"** So that it is a string. 

this is the button
```meta-bind-button
label: Filter References
icon: ""
hidden: false
class: ""
tooltip: ""
id: ""
style: default
actions:
  - type: updateMetadata
    bindTarget: ZK References#status
    evaluate: false
    value: "2001"

```

I tried with
value: 2001 - > status: 2001
value: "2001" -> status: 2001
value: '"2001"'  -> status: '"2001"'
value: '2001; -> status: 2001

is there any way to set value so that i get status: "2001"

When status: 2001, I get a datatype error in Obsidian.

**Comment by mProjectsCode** [comment, marked as answer]
Currently no. Everything that looks like a number gets converted to a number.

**Reply by FamedBear16** [reply]
Would it be possible to include this in a future release? The ability to pass a number as a string so that the property is status: "2021"
Thanks

**Reply by mProjectsCode** [reply]
Maybe, though I don't want to introduce breaking changes, so this might be a bit difficult.

**Comment by FamedBear16** [comment]
Hi @mProjectsCode ,
I just wanted to let you know that I sorted it out by using the INPUT Field and the exceptionally good metadata embedding.
 I managed to reach a solution that was way above my expectation thanks to your work. Congratulation for you programming skills.
